### PR TITLE
fix find_package(... GLOBAL) for CMake < 3.24

### PR DESCRIPTION
**Pull request by apwojcik**
The fix for the find_package(... GLOBAL) and use of CMake version lower than 3.24.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -403,6 +403,11 @@ option(BUILD_FILE_REORG_BACKWARD_COMPATIBILITY "Build with file/folder reorg wit
 set( MIOPEN_INSTALL_DIR miopen)
 set( DATA_INSTALL_DIR ${CMAKE_INSTALL_DATAROOTDIR}/miopen )
 
+if(MIOPEN_ENABLE_AI_KERNEL_TUNING OR MIOPEN_ENABLE_AI_IMMED_MODE_FALLBACK)
+    find_package(frugally-deep CONFIG REQUIRED)
```
Comment (atamazov) [inline CMakeLists.txt:407]: So we do not need 3.24 anymore? Good!

**Review by atamazov — APPROVED** [review]
LGTM

File: CMakeLists.txt (hunk 1 above)
Comment (apwojcik) [reply CMakeLists.txt:407]: Correct :-)

**Comment by atamazov** [comment]
Resolves #2607 and makes #2608 unnecessary.

**Comment by atamazov** [comment]
🟢 Works fine with ROCm 5.7.0 and cmake 3.16.3

**Comment by junliume** [comment]
Thank you for the quick fix! @apwojcik

**Review by junliume — APPROVED** [review]

**Pull request merged**